perfkitbenchmarker/linux_benchmarks/cassandra_ycsb_benchmark.py

```diff
@@ -50,10 +50,9 @@
 
 # TODO: Add flags.
 REPLICATION_FACTOR = 3
+COLUMN_FAMILY = 'data'
```

Comment:
It's used in line 165 and passed to YCSB execution params:
https://github.com/kamsz/PerfKitBenchmarker/blob/cassandra_ycsb/perfkitbenchmarker/linux_benchmarks/cassandra_ycsb_benchmark.py#L165